02LessonTwo/LessonTwo.js

```diff
@@ -1,14 +1,31 @@
 module.exports = function() {
 
-  this.whichSpecies = function (character) {
+this.whichSpecies = function (character) {
     // should return "dog" when character is 'scooby'
+    if (character === 'scooby') {
+    	return "dog";
+    }
     // should return "cat" when character is 'garfield'
+    else if (character === 'garfield') {
```

Comment:
This looks good. Can you explain whether the else if is required in this circumstance as opposed to a regular if?